src/intrinsics/react-mocks/mocks.js

```diff
@@ -85,22 +226,49 @@ let reactCode = `
         }
         props.children = childArray;
       }
+    
+      return ReactElement(element.type, key, ref, self, source, owner, props);
+    }
 
-      return {
-        $$typeof: element.$$typeof,
-        type: element.type,
-        key: key,
-        ref: ref,
-        props: props,
-        _owner: owner,
-      };
-    },
+    function isValidElement(object) {
+      return (
+        typeof object === 'object' &&
+        object !== null &&
+        object.$$typeof === REACT_ELEMENT_TYPE
+      );
+    }
+
+    return {
+      Children: {
+        forEach: forEachChildren,
+        map: mapChildren,
+        count: countChildren,
+        only: onlyChild,
+        toArray,
+      },
+      Component,
+      PureComponent,
+      createElement,
+      cloneElement,
+      isValidElement,
+    };
   }
 `;
 let reactAst = parseExpression(reactCode, { plugins: ["flow"] });
 
 export function createMockReact(realm: Realm): ObjectValue {
-  let reactValue = Environment.GetValue(realm, realm.$GlobalEnv.evaluate(reactAst, false));
+  let reactFactory = Environment.GetValue(realm, realm.$GlobalEnv.evaluate(reactAst, false));
+  invariant(reactFactory instanceof ECMAScriptSourceFunctionValue);
+
+  let currentOwner = (realm.react.currentOwner = new ObjectValue(
+    realm,
+    realm.intrinsics.ObjectPrototype,
+    "currentOwner"
+  ));
```

Comment:
This should have `current: null`. How does this interact with the real ReactCurrentOwner? Does this mean Prepacked createElement calls will always have no owner? Or is there some way that we should connect it to the real one?

Reply:
It's not needed right now as `current` is never accessed at this point. I'm creating the object to be used later.